lnbits/core/views/api.py

```diff
@@ -677,6 +686,41 @@ async def api_auditor():
     }
 
 
+@core_app.post("/api/v1/push_notification", status_code=HTTPStatus.CREATED)
+async def api_create_push_notification_subscription(
+    request: Request,
+    data: CreatePushNotificationSubscription,
+    wallet: WalletTypeInfo = Depends(require_admin_key),
+):
+    subscription = json.loads(data.subscription)
+    endpoint = subscription["endpoint"]
+    host = urlparse(str(request.url)).netloc
+    wallet_ids = [wallet.wallet.id]
+
+    user = await get_user(wallet.wallet.user)
+    if user:
+        wallet_ids = user.wallet_ids
+
+    for wallet_id in wallet_ids:
+        new_subscription = await create_push_notification_subscription(
+            endpoint, wallet_id, data.subscription, host
+        )
+        assert (
+            new_subscription
+        ), "Newly created push notification subscription couldn't be retrieved"
+
+    return ""
```

Comment:
usually we return the model on a POST request, any reason not to do that here? if not also add some return type